example/01_point_example.cpp

```diff
@@ -107,6 +107,29 @@ int main()
 
 
     // Some ways of getting point values
+    typedef model::point<double, 3, cs::cartesian> point_t;
+    typedef model::ring<point_t> ring_t;
+    typedef model::PolyhedralSurface<ring_t> poly_t;
```

Comment:
Since the library is now C++14, the aliases could be declared with the `using` declarations, which is a bit neater, IMHO, as it puts the most important part, the name, to the left.